internal/services/containerapps/container_app_environment_resource.go

```diff
@@ -191,7 +198,8 @@ func (r ContainerAppEnvironmentResource) Create() sdk.ResourceFunc {
 							SharedKey:  keys.Model.PrimarySharedKey,
 						},
 					},
-					VnetConfiguration: &managedenvironments.VnetConfiguration{},
+					DaprAIConnectionString: pointer.To(containerAppEnvironment.DaprAIConnectionString),
```

Comment:
Can we move this out of the block here and only set when there is a value in config?

Reply:
Sure.